CHANGES.rst

```diff
@@ -1,6 +1,15 @@
 Changes
 =======
 
+- New ``filter`` argument for :func:`typestats`, :func:`most_common_types`,
+  :func:`show_most_common_types`, :func:`show_growth`.
+
+- Show lambda function more human-readble with change to :func:`_short_repr`
+
+
+3.1.3 (unreleased)(2017-12-19)
+------------------
+
```

Comment:
Thank you for these!  But you accidentally duplicated the 3.1.3 version header.  Also, the new entries should be below the header (replace the existing "- Nothing changed yet." text).  And there's no need to add a date -- I'll add it when I make the release.